pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -10814,7 +10814,7 @@
         title: "How to Host an Hour"
         desc: "A step-by-step guide to preparing for, running, and celebrating your Hour of Code event with your class or group."
         button: "Read the guide"
-      host:
```

Comment:
Is it going to cause any issues to change this key?

Reply:
I don't think so, if I remember correctly there was a similar instance of this before and Mario and Jorge said it's ok to change the key.

Reply:
My one concern is that we've got such a condensed timeline but I think this makes sense